### ISSUE TITLE
Latest release broke builds

### ISSUE THREAD

**Issue by cart** [KyleMayes/clang-sys#154]
We're now hitting the following error message in our macos builds over in the Bevy repo:

```txt
error[E0425]: cannot find function `clang_CXXMethod_isVirtual` in this scope
    --> /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.61.0/clang.rs:817:18
     |
817  |           unsafe { clang_CXXMethod_isVirtual(self.x) != 0 }
     |                    ^^^^^^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `clang_CXXMethod_isPureVirtual`
     |
    ::: /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/clang-sys-1.5.1/src/lib.rs:1735:1
     |
1735 | / link! {
1736 | |     pub fn clang_CXCursorSet_contains(set: CXCursorSet, cursor: CXCursor) -> c_uint;
1737 | |     pub fn clang_CXCursorSet_insert(set: CXCursorSet, cursor: CXCursor) -> c_uint;
1738 | |     pub fn clang_CXIndex_getGlobalOptions(index: CXIndex) -> CXGlobalOptFlags;
...    |
2289 | |     pub fn clang_VerbatimLineComment_getText(comment: CXComment) -> CXString;
2290 | | }
     | |_- similarly named function `clang_CXXMethod_isPureVirtual` defined here
```

This appears to be tied to the latest release of `clang-sys`.

https://github.com/bevyengine/bevy/actions/runs/4099201878/jobs/7068888325

**Comment by cart** [comment]
Presumably, this pushed a breaking change in a minor release, which got picked up by current `bindgen` releases?

**Comment by mockersf** [comment]
probably from https://github.com/KyleMayes/clang-sys/commit/d845dabb99e38c73e0c4a9dfa3f8e8df1db5ebb0

**Comment by cart** [comment]
I am of the mind that this release should be immediately yanked until this can be sorted out. This is going to affect a lot of people:
![image](https://user-images.githubusercontent.com/2694663/216858377-b2d1a1e5-c93f-46b9-a127-2f53b7a687ed.png)
(the bindgen crate)

**Comment by KyleMayes** [comment]
Oops.

Yanked.

This will get fixed eventually.
Or never, this crate is way more trouble than its worth to me.

**Comment by cart** [comment]
Much appreciated. Thank you :)

**Comment by KyleMayes** [comment]
Alright, this is (hopefully) fixed.
I've released `v1.6.0` that re-added the accidentally deleted code with the changes from `v1.5.0` and `v1.5.1`.

To prevent this from happening again, I've also added a test crate to this repo that compiles and uses `bindgen` with the local version of `clang-sys` and added a step to the CI job to compile and run this crate. That should hopefully prevent accidental `bindgen` breakages from getting through in the future.
